project/Build.scala

```diff
@@ -1713,26 +1713,26 @@ object Build {
         scalaVersion.value match {
           case Default2_11ScalaVersion =>
             Some(ExpectedSizes(
-                fastLink = 516500 to 517500,
-                fullLink = 107000 to 108000,
-                fastLinkGz = 65500 to 66500,
-                fullLinkGz = 28000 to 29000,
+                fastLink = 379000 to 38000,
```

Comment:
Ach, the CI is complaining here:
```suggestion
                fastLink = 379000 to 380000,
```